### DISCUSSION TITLE
Sync is failing

### DISCUSSION THREAD

**Discussion by NeuroDawg** [CyferShepard/Jellystat#45] [Q&A] [answered]
I've installyed Jellystat via docker (wondefrul program, btw) but my sync is failing. If someone could help me figure out why that would be great. Here's the jellystat log:

> Error performing bulk insert:Bulk insert error: Error: error: null value in column "CollectionType" of relation "jf_libraries" violates not-null constraint
> Syncing... 1/3
> Beginning Library Item Sync
> Error performing bulk insert:Bulk insert error: Error: error: insert or update on table "jf_library_items" violates foreign key constraint "jf_library_items_parentid_foreign"
> 0 Library Items Removed.
> Item Sync Complete
> Syncing... 2/3
> Beginning Seasons and Episode sync
> Seasons: 0 Rows Inserted. 0 Rows Updated.
> 0 Seasons Removed.
> Episodes: 0 Rows Inserted. 0 Rows Updated.
> 0 Episodes Removed.
> Sync Complete
> Syncing... 3/4
> Beginning File Info Sync
> 0 Item Info inserted.
> 0 Item Info Removed.
> 0 Episodes Info inserted.
> 0 Episodes Info Removed.
> Info Sync Complete
> Syncing... 4/4
> Removing Orphaned FileInfo/Episode/Season Records
> Orphaned FileInfo/Episode/Season Removed.
> Sync Complete

Thanks!

**Comment by CyferShepard** [comment]
Hi @NeuroDawg , im currently looking into this, should have a fix by tomorrow

**Comment by CyferShepard** [comment]
@NeuroDawg  would it be possible to take a look at your library folder overview? its in your dashboard in jellyfin under Libraries

**Reply by NeuroDawg** [reply]
Here you go. If you need any other information, let me know.

![JF Libraries](https://github.com/CyferShepard/Jellystat/assets/59210675/9a010ad3-a1fd-4bec-b298-d31facfbc4e1)

**Comment by CyferShepard** [comment, marked as answer]
Hi @NeuroDawg , a fix has been pushed, should be available in a few minutes, let me know if that helps

**Reply by NeuroDawg** [reply]
It's syncing now! Thanks.

But, it's not collecting all data. My movies are severely undercounted and no play/activity/user data is loaded. I'll keep an eye out for updates and keep working with it.
